src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs

```diff
@@ -13,11 +13,17 @@ public enum SyntaxKind : ushort
         List = GreenNode.ListKind,
 
         // punctuation
+        /// <summary>Represents bitwise complement operator.</summary>
         TildeToken = 8193,
+        /// <summary>Represents either a not logical operator or a null-forgiving operator.</summary>
```

Comment:
The enum will need a very careful review. Some tokens like ExclamationToken may have a few different meanings. So keep in mind any other possible meanings of the token when reviewing.